main.py

```diff
@@ -0,0 +1,37 @@
+import asyncio
+import grpc
+from src.grpc.server import UserManagement
+
+
+from src.db.database import Database
+from src.grpc.generated import user_pb2_grpc
+
+DB_URL = "postgresql+asyncpg://test:test123@postgres:5432/user_management"
+
+
+async def init_db():
```

Comment:
This is no longer the way we do things. Database should be initialized only once outside of code, use migrations.